src/cbmc/bmc.h

```diff
@@ -193,6 +193,10 @@ class bmct:public safety_checkert
 
   friend class bmc_all_propertiest;
 
+  // \todo remove these two declarations once test-gen dependency is fixed
+  bool cover(const goto_functionst &goto_functions);
```

Comment:
How does adding the declaration (but not a definition) make a difference?

Reply:
Heh... wondered the same thing - I think the answer is we provide the definition in the test-gen repo. I vaguely remember some dark magic where we linked a _different_ `bmc_cover.cpp` because it had diverged too much on the old branch...